testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPoliciesTest.java

```diff
@@ -2732,6 +2736,255 @@ public void testExtendedClientPolicyIntefacesForDeviceTokenRequest() throws Exce
         assertEquals("Exception thrown intentionally", tokenResponse.getErrorDescription());
     }
 
+    @Test
+    public void testExtendedClientPolicyIntefacesForDeviceTokenResponse() throws Exception {
```

Comment:
Note: Looks that ClientPoliciesTest class is already very big and contains 49 test methods. Will it makes sense to add the new test for "extended"  methods into separate test class?

Reply:
Yes, I also think that `ClientPoliciesTest` became too big. How about creating other PR for refactoring `ClientPoliciesTest`? I am willing to work on this refactoring.

Reply:
@tnorimat +1, if you can create follow-up GH issue and the follow-up PR, it will be very nice. Thank you!